pkg/resources/cruisecontrol/deployment.go

```diff
@@ -203,6 +205,19 @@ func GeneratePodAnnotations(kafkaCluster *v1beta1.KafkaCluster, capacityConfig s
 	return util.MergeAnnotations(annotations...)
 }
 
+func getInitResourcesRequirements() corev1.ResourceRequirements {
```

Comment:
Let's move this function to a common place in order to not duplicate it. Also I'd rename it to something like `getDefaultInitContainerResourcesRequirements` as that name better reflects what it is for.